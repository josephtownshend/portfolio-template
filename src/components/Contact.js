import React from 'react';
import '../App.css';

export default class Contact extends React.Component {

  render() {
    return(
      <div>email@gallery.com</div>
    )
  }
}
