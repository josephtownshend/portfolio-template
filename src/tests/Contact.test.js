import React from 'react';
import { configure } from 'enzyme';
import { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';
import renderer from 'react-test-renderer';
import Contact from '../components/Contact';

configure({ adapter: new Adapter() });

it('shallow renders without crashing', () => {
  shallow(<Contact />)
});

it('renders correctly', () => {
  const tree = renderer.create(<Contact />).toJSON();
  expect(tree).toMatchSnapshot();
});
