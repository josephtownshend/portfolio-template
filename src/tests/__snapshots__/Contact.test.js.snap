// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div>
  <p
    className="BlockText"
  >
    For all sales enquires please get in touch with 
    <a
      href="mailto: info@gallery.com"
    >
      Gallery Gallery
    </a>
    .
  </p>
  <p
    className="BlockText"
  >
    For all other enquires please get in touch 
    <a
      href="mailto: info@personal.com"
    >
      directly
    </a>
    .
  </p>
  <br />
  <p
    className="BlockText"
  >
    My Artist Studio 
    <br />
    123 Studio 
    <br />
    The Studio Building
    <br />
    London
    <br />
    SE15 123
  </p>
</div>
`;
